api/test/unit/services/application.service.spec.ts

```diff
@@ -89,7 +89,8 @@ export const mockApplication = (position: number, date: Date) => {
       },
     },
     demographics: {
-      race: ['declineToRespond'],
```

Comment:
Should this `howDidYouHear` be showing up in the export above in the test, or are those not connected?